src/EmbeddedSonarAnalyzer.props

```diff
@@ -7,7 +7,13 @@
     <EmbeddedSonarAnalyzerVersion>8.51.0.59060</EmbeddedSonarAnalyzerVersion>
     <EmbeddedSonarCFamilyAnalyzerVersion>6.41.0.60884</EmbeddedSonarCFamilyAnalyzerVersion>
     <EmbeddedSonarJSAnalyzerVersion>9.12.0.20319</EmbeddedSonarJSAnalyzerVersion>
-    <EmbeddedSonarSecretsAnalyzerVersion>1.0.3</EmbeddedSonarSecretsAnalyzerVersion>
+	
+	<!-- Secrets
+	     Secrets are special case: we get the analyzer binaries from a NuGet package, and the
+		 rule help from an "normal" Java plugin. The two artefacts are versioned separately.
+	-->
+    <EmbeddedSonarSecretsNuGetVersion>1.0.3</EmbeddedSonarSecretsNuGetVersion>
```

Comment:
FYI the secrets NuGet package is now being published to a Repox feed and version 1.0.4 has just bee released, albeit with no code changes.

I've created #3710 to handle changing the package feed/version.